### PR TITLE
SPLAT-1007 Release v3.14.2

### PR DESCRIPTION
**Pull request by bender-wk**
JIRA: https://jira.atl.workiva.net/browse/SPLAT-1007

@Workiva/product2

* #1468 Java: Bump checkstyle from 8.40 to 8.41 in /lib/java
* #1474 Java: Bump jmh.version from 1.27 to 1.28 in /lib/java
* #1467 Python: Bump coverage from 5.4 to 5.5 in /lib/python
* #1484 Examples: Bump activemq-client from 5.15.11 to 5.16.1 in /examples/java
* #1487 Java: Bump junit from 4.13.1 to 4.13.2 in /lib/java
* #1486 Java: Bump netty-all from 4.1.59.Final to 4.1.60.Final in /lib/java
* #1491 Tests: Bump netty-all from 4.1.59.Final to 4.1.60.Final in /test/integration/java/frugal-integration-test
* #1483 Examples: Bump netty-all from 4.1.59.Final to 4.1.60.Final in /examples/java
* #1492 Tests: Bump junit from 4.13.1 to 4.13.2 in /test/integration/java/frugal-integration-test
* #1466 Java: Bump mockito-core from 3.7.7 to 3.8.0 in /lib/java
* #1493 Tests: Bump activemq-client from 5.15.11 to 5.16.1 in /test/integration/java/frugal-integration-test
* #1497 Java: Bump libthrift from 0.14.0 to 0.14.1 in /lib/java
* #1489 Go: Bump github.com/sirupsen/logrus from 1.8.0 to 1.8.1 in /lib/go
* #1510 go-exam: bump github.com/go-stomp/stomp from 2.1.2+incompatible to 2.1.3+incompatible in /examples/go
* #1502 Go: Bump github.com/go-stomp/stomp from 2.1.2+incompatible to 2.1.3+incompatible in /lib/go
* #1509 go-tool: bump github.com/sirupsen/logrus from 1.8.0 to 1.8.1
* #1501 Python: Bump flake8 from 3.8.4 to 3.9.0 in /lib/python
* #1490 Go: Bump github.com/apache/thrift from 0.14.0 to 0.14.1 in /lib/go
* #1512 go-exam: bump github.com/sirupsen/logrus from 1.8.0 to 1.8.1 in /examples/go
* #1508 go-tool: bump github.com/urfave/cli from 1.19.1 to 1.22.5
* #1503 Go: Bump github.com/nats-io/nats-server/v2 from 2.1.9 to 2.2.0 in /lib/go
* #1514 go-test: bump github.com/sirupsen/logrus from 1.7.0 to 1.8.1 in /test/integration
* #1511 go-exam: bump github.com/apache/thrift from 0.14.0 to 0.14.1 in /examples/go
* #1516 java-exam: bump libthrift from 0.13.0 to 0.14.0 in /examples/java
* #1519 go-tool: bump gopkg.in/yaml.v2 from 2.2.2 to 2.4.0
* #1528 fix: revert all changes from PR #1476

### CODE REVIEW

**Comment by aviary3-wk** [comment]
## Security Insights
<details>
<summary>(3) Vulnerable direct dependencies were detected</summary>
<li>1 vulns in <code>aiohttp < 3.7.4</code> via <code>lib/python/requirements_dev_asyncio.txt</code></li>
<li>1 vulns in <code>org.apache.thrift:libthrift < 0.14.0</code> via <code>examples/java/pom.xml</code></li>
<li>1 vulns in <code>org.apache.thrift:libthrift < 0.14.0</code> via <code>test/integration/java/frugal-integration-test/pom.xml</code></li>
</details>

## Action Items
* Review dependencies for available updates
* See <a href="https://splunk.workiva.net/en-US/app/search/security_vulnerable_dependencies?form.repository=frugal">this Splunk dashboard</a> for more CVE details
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://dev.workiva.net/docs/teams/information-security/aviary#configuring-aviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: lib/go/go.mod [hunk 1]
```diff
@@ -3,11 +3,11 @@ module github.com/Workiva/frugal/lib/go
 go 1.15
 
 require (
-	github.com/apache/thrift v0.14.0
-	github.com/go-stomp/stomp v2.1.2+incompatible
-	github.com/nats-io/nats-server/v2 v2.1.9
-	github.com/nats-io/nats.go v1.10.0
+	github.com/apache/thrift v0.14.1
```
Comment (houstonking-wf) [inline lib/go/go.mod:6]: Note: bumping to v0.14.1 for go

File: lib/java/pom.xml [hunk 2]
```diff
@@ -50,7 +50,7 @@
         <dependency>
             <groupId>org.apache.thrift</groupId>
             <artifactId>libthrift</artifactId>
-            <version>0.14.0</version>
+            <version>0.13.0</version>
```
Comment (houstonking-wf) [inline lib/java/pom.xml:53]: Note: here's the downgrade to 0.13.0 for java.

File: examples/java/pom.xml [hunk 3]
```diff
@@ -17,17 +17,17 @@
         <dependency>
             <groupId>org.apache.thrift</groupId>
             <artifactId>libthrift</artifactId>
-            <version>0.13.0</version>
+            <version>0.14.0</version>
```
Comment (houstonking-wf) [inline examples/java/pom.xml:20]: Note: this probably isn't desirable, but we've got a backlog of repairs to do on the examples anyway.

**Review by houstonking-wf — APPROVED** [review]

**Comment by houstonking-wf** [comment]
QA +1; tests are passing on my PR -- it's a direct revert of another PR.

**Comment by houstonking-wf** [comment]
@Workiva/release-management-p for merge

**Review by rmconsole-wf — APPROVED** [review]
+1 from RM

**Pull request merged**